test/type_traits.cpp

```diff
@@ -723,6 +723,70 @@ TEST_CASE("void_t", "[meta.trans.other]") {
   CHECK(std::is_same<slb::void_t<int, float, double>, void>::value);
 }
 
+// 23.15.8, logical operator traits
+
+struct True {
+  static constexpr bool value = true;
+};
+
+struct False {
+  static constexpr bool value = false;
+};
+
+// template<class... B> struct conjunction;
+TEST_CASE("conjunction", "[meta.logical]") {
+  CHECK(slb::conjunction<>::value == true);
+
+  CHECK(slb::conjunction<slb::true_type>::value == true);
+  CHECK(slb::conjunction<slb::false_type>::value == false);
+
+  CHECK(slb::conjunction<True>::value == true);
+  CHECK(slb::conjunction<False>::value == false);
+
+  CHECK(slb::conjunction<slb::true_type, slb::true_type>::value == true);
+  CHECK(slb::conjunction<slb::true_type, slb::false_type>::value == false);
+  CHECK(slb::conjunction<slb::false_type, slb::true_type>::value == false);
+  CHECK(slb::conjunction<slb::false_type, slb::false_type>::value == false);
+
+  CHECK(slb::conjunction<True, True>::value == true);
+  CHECK(slb::conjunction<True, False>::value == false);
+  CHECK(slb::conjunction<False, True>::value == false);
+  CHECK(slb::conjunction<False, False>::value == false);
+}
```

Comment:
Needs tests for short-circuiting behavior, as well as for `Bi` being a base class.